test/RyanJsonRFC8259JsonTest.c

```
@@ -0,0 +1,613 @@
+	// 	cJSON_Minify(data);
+	// 	if (0 != strcmp(data, str)) { printf("-- 原始: %s -- 序列化: %s\n", data, str); }
+
+	// 	free(str);
+	// #endif
+
+	// 	yyjson_doc_free(doc);
+	// 	return 0;
+	// err:
+	// 	yyjson_doc_free(doc);
+	// 	return -1;
+}
```

Cppcheck (2.18.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)